app/src/main/res/values-uz/strings.xml

```diff
@@ -90,6 +92,8 @@
     </string>
   <string name="search_hint">Izlash</string>
   <string name="quranSearchType">Suralar</string>
+  <string name="search_full_results">Barcha natijalar</string>
```

Comment:
I could not find use case for this string (and one below). From the source logic, it looks like it is only used in the Arabic locale. Maybe put the Arabic translations in the default locale strings and make them non-translatable? What do you think @ahmedre?

Reply:
yeah that sounds reasonable today - but am wondering if we should have something similar for non-Arabic cases where the number of results is a lot more than what we'd show in the suggestions instead.